src/nanosvg.h

```diff
@@ -2578,6 +2591,29 @@ static void nsvg__parseGradientStop(NSVGparser* p, const char** attr)
 	stop->offset = curAttr->stopOffset;
 }
 
+static void nsvg__parseGroup(NSVGparser* p, const char** attrs)
+{
+	NSVGgroup* group, *g;
+	NSVGattrib* attr = nsvg__getAttr(p);
+	
+	nsvg__parseAttribs(p, attrs);
+	if (!*attr->id) //skip anonymous groups
```

Comment:
Make this a bit more obvious:
```
if (attr->id[0] == '\0')
```
If the idea of storing the groups is to store the hierarchy, I think it would make sense to anonymous groups too.